elastica/joint.py

```diff
@@ -209,9 +210,14 @@ class FixedJoint(FreeJoint):
             Damping coefficient of the joint.
         kt: float
             Rotational stiffness coefficient of the joint.
+        nut: float
+            Rotational damping coefficient of the joint.
+        static_rotation: np.array
+            Static 3x3 rotation matrix from system one to system two.
+            Instead of aligning the directors of both systems directly, a desired rotational offset C_12* is enforced.
     """
 
-    def __init__(self, k, nu, kt):
+    def __init__(self, k, nu, kt, nut=0.0, use_static_rotation=True):
```

Comment:
```suggestion
    def __init__(self, k, nu, kt, nut=0.0, is_rest_joint_state=True):
```

Reply:
Resolved in https://github.com/GazzolaLab/PyElastica/pull/135/commits/9a5cf1cb90ebd7fb6a215d799eb27c5d0bd1c5f0